content/app/development/configuration/process/message/_index.en.md

```diff
@@ -2,22 +2,20 @@
 title: Message
 description: How to set up a message in Altinn Studio.
 toc: true
-tags: [translate-to-english]
 ---
 
-En melding i Altinn 3 er egentlig bare et data-steg, på samme måte som f.eks. skjemautfylling. Det settes opp med en datamodell for meldingen, og en 
-layout for hva som skal vises på siden. Melding er dermed ikke en egen steg-type. Dette gjør at en melding i Altinn 3 er ekstremt fleksibel, og kan settes opp enten som eneste steg i en prosess, eller som en del av en større prosess.
+A message in Altinn 3 is really just a data step in the same way as for eks. form filling. It is set up through a datamodel for the message, and a layout 
+for what the message should look like. Because of this, message is not its own task type. This enables greater flexibility for Messages in Altinn 3 and allows them to be used as either the only task in a process, or as a part of a larger process.
 
-Vi har lagd noen verktøy som skal gjøre det enkelt å komme i gang med å sette opp en melding i en app.
+We have made some tools to simplify the process of getting started with setting up a message in an app.
 
-## Datamodell
-Vi har lagd en standard datamodell for meldinger, for å gjøre det enkelt å komme i gang. Denne datamodellen kan man finne [her](https://altinncdn.no/schemas/xsd/message/message.schema.v1.xsd). Denne kan enten brukes som den er, brukes som et utgangspunkt, eller man kan bruke en helt annen datamodell. 
+## Datamodel
+We have made a standard datamodel for messages, to make getting started easier. This datamodel can be found [here](https://altinncdn.no/schemas/xsd/message/message.schema.v1.xsd). It can be used as is, as a starting point, or exchanged for your own datamodel.  
 
 ## Layout
-Layout'en kan man definere helt selv, på samme måte som for skjema. Vi har allikevel opprettet en [meldings-widget](../../../ux/ui-editor/widgets), for å gjøre det enklere å komme i gang. Denne widget'en inneholder alle komponentene som trengs for å lage visningen under. Den inneholder også tekster som legges til i ressurs-filene automatisk, som har [variabler](../../../ux/tekster/#variabler-i-tekster) med referanser til feltene `Title` og `Body` i standard datamodell. Dersom man ønsker andre tekster eller å bruke en annen datamodell, er det bare å redigere enten komponentene eller tekstene etter ønske etter at de er lagt inn i siden.
+The layout can be defined by yourself in the same way as a form. We have also created a [message widget](), to make it easier to get started. This widget contains all the necessary components to recreate the example below. It also contains texts which are added to the resource files automatically, and contains [variables]() with referances to `Title` and `Body` in a standard datamodel. If you should wish for different texts, or to use a different datamodel, just edit either the components or the texts according to your wishes after they have been added to the page.
```

Comment:
```suggestion
The layout can be defined by yourself in the same way as a form. We have also created a [message widget](), to make it easier to get started. This widget contains all the necessary components to recreate the example below. It also contains texts which are added to the resource files automatically, and contains [variables]() with references to `Title` and `Body` in a standard data model. If you should wish for different texts, or to use a different data model, just edit either the components or the texts according to your wishes after they have been added to the page.
```

Seems like you also forgot to fix the links here? They won't work without a target.. :smile: If the target pages are not translated yet, or have been translated in another PR, try to make sure the links will work when all PRs are merged.